config/config.go

```diff
@@ -876,15 +876,19 @@ type MempoolConfig struct {
 	// Including space needed by encoding (one varint per transaction).
 	// XXX: Unused due to https://github.com/tendermint/tendermint/issues/5796
 	MaxBatchBytes int `mapstructure:"max_batch_bytes"`
-	// Experimental parameter to limit broadcast of txs to up to this many peers.
-	// If we are connected to more than this number of peers, only send txs to the first
-	// ExperimentalMaxUsedOutboundPeers of them. If one of those peers disconnects, activate another
-	// peer.
-	// If set to 0, this feature is disabled, that is, the number of active connections is not
-	// bounded.
-	// If enabled, a value of 10 is recommended based on experimental performance results using the
-	// default P2P configuration.
-	ExperimentalMaxUsedOutboundPeers int `mapstructure:"experimental_max_used_outbound_peers"`
+	// Experimental parameters to limit gossiping txs to up to the specified number of peers.
+	// We use two independent upper values for persistent peers and for non-persistent peers.
+	// Unconditional peers are not affected by this feature.
+	// If we are connected to more than the specified number of persistent peers, only send txs to
+	// the first ExperimentalMaxGossipConnectionsToPersistentPeers of them. If one of those
```

Comment:
This seems like it should not be the first 10, but instead a random 10 peers, no?